client/src/pages/Note/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+
+import Button from '../../components/Button'
+
+import './style.css'
+
+export default class Page2 extends Component {
+
+  state = {
+    guidlinesApproved: false,
+    checked: false
```

Comment:
This would be the hooks equilivant 

const [guidlinesApproved, setGuidlinesApproved] = React.useState(false);
const [checked, setChecked] = React.useState(false);

Reply:
we actually could just use one state because they will always be the same value

Reply:
We can make a refactor for the code after this weekend so i can learn the hooks.
Not the same value because i make the text clickable too, so i need to change the state when the user clicks on the text or the checkbox
@dubhcait

Reply:
if you label tag instead of span, it'll have the same functionality as that, the attribute "for" is key thought

Reply:
I add a label tag instead of span